packages/next/next-server/server/load-components.ts

```diff
@@ -64,6 +64,8 @@ export async function loadComponents(
   const DocumentMod = require(documentPath)
   const { middleware: DocumentMiddleware } = DocumentMod
 
+  const AppMode = require(appPath)
```

Comment:
This is server-side only code so I'm guessing it doesn't need changed?

Reply:
@Timer,
You are right, this change is not needed to fix the css issue I described above, but since the import/require order can affect side effects started on root components I thought it was a good idea to change the import order on this point as well.

Reply:
OK -- we need to add tests for both of these modified behaviors then.

Reply:
@Timer,
pushed tests for these new beheaviors.